### DISCUSSION TITLE
Failure to install via msi on LTSC Win10

### DISCUSSION THREAD

**Discussion by AbyssalMonkey** [Difegue/LANraragi#557] [Q&A] [answered]
**LRR Version and OS**
.8.3 Windows 10 LTSC (1809)

**Bug Details**
Fails to install, stops at cmd. Had .7.8 working, decided to update, and fails.

**Matching Logs**
[install.log](https://github.com/Difegue/LANraragi/files/7611052/install.log)

**Screenshots**

**Comment by Difegue** [comment, marked as answer]
Version 0.8.0 has broken compatibility with LTSC versions of Windows 10, as specified in the [documentation](https://sugoi.gitbook.io/lanraragi/v/dev/installing-lanraragi/methods#windows-10-11-lrr-for-windows) and [release notes](https://github.com/Difegue/LANraragi/releases/tag/v.0.8.0):  

> Since LRR 0.8.0, you need Windows 10 version 1903 at least. 0.7.9 will work with version 1809 if you're on an LTSC channel.  

This was done to make for a better installation experience (and to eventually auto-install WSL if needed, although I haven't gotten to that yet), although you should be able to use the latest versions with some manual tinkering:  

- Install 0.8.3 so you have a Karen/bootstrapper install (not sure if those are removed on install failures, if they do install 0.7.9 beforehand I guess)
- Get the `package.tar` WSL distro for the version you want to install: 7-zip can pull them out of .MSI files easily.
- Install the distro manually using [LxRunOffline](https://github.com/DDoSolitary/LxRunOffline/releases) since 1809 has no easy support for sideloading .tars on its own. You can install the distro in any folder you want, but it **has** to be named `lanraragi` in order to be picked up by the bootstrapper.

**Reply by Difegue** [reply]
Just noticed I'd already answered this in https://github.com/Difegue/LANraragi/discussions/522 but meh, this reply is more complete ¯\\\_(ツ)_/¯

**Reply by AbyssalMonkey** [reply]
> Version 0.8.0 has broken compatibility with LTSC versions of Windows 10, as specified in the [documentation](https://sugoi.gitbook.io/lanraragi/v/dev/installing-lanraragi/methods#windows-10-11-lrr-for-windows) and [release notes](https://github.com/Difegue/LANraragi/releases/tag/v.0.8.0):
> 
> > Since LRR 0.8.0, you need Windows 10 version 1903 at least. 0.7.9 will work with version 1809 if you're on an LTSC channel.

This might be something to add to the page here: https://sugoi.gitbook.io/lanraragi/installing-lanraragi/windows
